ServerCore/Pages/Teams/Delete.cshtml.cs

```diff
@@ -37,7 +37,7 @@ public async Task<IActionResult> OnPostAsync(int teamId)
                 await TeamHelper.DeleteTeamAsync(_context, Team);
             }
 
-            return RedirectToPage("./Index");
+            return RedirectToPage("./List");
```

Comment:
is this destination right for admins as well? Other places have needed to check which EventRole is active and I'm not sure if this is one of them.

Reply:
Changed to send admins to index (list works too, but index is more useful)